index.html

```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
         </section>
 
         <footer>
-            <div class="remarks">
-                <p>
-                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-                </p>
+            <div class="last-part">
+                <div class="remarks">
+                    <p>
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>
+                </div>
+    
+                <form method="POST" action="https://formspree.io/f/xzbqlowk" class="footer-form">
+                    <div class="full-name">
+                        <input type="text" name="first name" maxlength="30" placeholder="First name" class="first-name">
+                        <input type="text" name="last name" maxlength="30" placeholder="Last name" class="last-name">
+                    </div>
+                    <input type="text" name="full name" maxlength="30" placeholder="Full name" class="name" required>
+                    <input type="email" name="email" placeholder="Email address" class="email" required>
+                    <textarea name="comments" cols="30" rows="5" maxlength="500" placeholder="Hey I’d like to connect to..."></textarea>
+                    <input type="submit" value="Get in touch" class="get-button">
+                    <p>wilsanfourrspeed@gmail.com</p>
+                </form>
+            </div>
+            <div class="footer-objects">
+                <img src="shape3.svg" alt="shape3" class="shape3">
+                <img src="shape4.svg" alt="shape4" class="shape4">
+                <img src="shape5.svg" alt="shape5" class="shape5">
             </div>
 
-            <form method="POST" action="https://formspree.io/f/xzbqlowk" class="footer-form">
-                <input type="text" name="full name" maxlength="30" placeholder="Full name" class="name" required>
-                <input type="email" name="email" placeholder="Email address" class="email" required>
-                <textarea name="comments" cols="30" rows="5" maxlength="500" placeholder="Hey I’d like to connect to..."></textarea>
```

Comment:
- [ ] Kindly fix the following to have a pixel perfect design.

- Your design
<img width="946" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/47b0068b-9723-4450-a4d0-21e6cdacbaed">

- Figma design
<img width="393" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/d9058455-71a3-4138-a336-7f14e2c6865a">

- The text on the left hand side is meant to be centered across the form element.
- The form input and textareas are supposed to be wider than that.
- The background images aren't properly fixed, please use the `background` css property position them properly using the css `background-position` property, and other background properties, you can make your research.
- The button `Get in touch` shouldn't be in the center.
- Add spacing between `First name` and `Last name` input tags.